README.md

```diff
@@ -1,61 +1,49 @@
+## What is Yomitan?
 
-Since the owner requested forks be uniquely named, we have chosen a new name, _yomitan_. (_-tan_ is an honorific used for anthropomorphic moe characters.) While we've made some substantial changes, the majority of the extension's functionality is thanks to hard work of foosoft and numerous other open source contributors from 2016-2023.
+Yomitan turns your web browser into a tool for building Japanese language literacy by helping you to **read** texts which would otherwise be too difficult to tackle.
 
-## Contributing
-
-Since this is a distributed effort, we **highly welcome new contributors**! Feel free to browse the [issue tracker](https://github.com/themoeway/yomitan/issues), and read our [contributing guidelines](./CONTRIBUTING.md). If you want to add or improve support for a language, read the documentation on [language features](./docs/development/language-features.md). You can also find us on [TheMoeWay Discord](https://discord.gg/UGNPMDE7zC) at [#yomitan-development](https://discord.com/channels/617136488840429598/1081538711742844980).
-
-## What's Yomitan?<!-- omit from toc -->
-
-Yomitan turns your web browser into a tool for building Japanese language literacy by helping you to decipher texts
-which would be otherwise too difficult to tackle. This extension is similar to [10ten Japanese Reader (formerly Rikaichamp)](https://addons.mozilla.org/en-US/firefox/addon/10ten-ja-reader/) for Firefox and [Rikaikun](https://chrome.google.com/webstore/detail/rikaikun/jipdnfibhldikgcjhfnomkfpcebammhp) for Chrome, but it stands apart in its goal of being an all-encompassing learning tool as opposed to a mere browser-based dictionary.
+This extension is similar to [10ten Japanese Reader (formerly Rikaichamp)](https://addons.mozilla.org/en-US/firefox/addon/10ten-ja-reader/) for Firefox and [Rikaikun](https://chrome.google.com/webstore/detail/rikaikun/jipdnfibhldikgcjhfnomkfpcebammhp) for Chrome.
 
 Yomitan provides advanced features not available in other browser-based dictionaries:
 
-- Interactive popup definition window for displaying search results.
-- On-demand audio playback for select dictionary definitions.
-- Kanji stroke order diagrams are just a click away for most characters.
-- Custom search page for easily executing custom search queries.
-- Support for multiple dictionary formats including [EPWING](https://ja.wikipedia.org/wiki/EPWING) via the [Yomitan Import](https://github.com/themoeway/yomitan-import) tool.
-- Automatic note creation for the [Anki](https://apps.ankiweb.net/) flashcard program via the [AnkiConnect](https://foosoft.net/projects/anki-connect) plugin.
-- Clean, modern code makes it easy for developers to [contribute](https://github.com/themoeway/yomitan/blob/master/CONTRIBUTING.md) new features.
+- 📈 Interactive popup definition window for displaying search results.
+- 🔊 Built-in audio playback for select Japanese words.
+- ✍ Kanji stroke order diagrams are generally just a click away.
```

Comment:
Not sure why the base unicode writing hand was used instead of the emoji version.
```suggestion
- ✍️ Kanji stroke order diagrams are generally just a click away.
```